package.json

```
@@ -2,36 +2,26 @@
   "name": "range-specifier-parser",
   "version": "1.0.2",
   "description": "Range Specifier Parser",
-  "homepage": "https://github.com/seegno/range-specifier-parser",
+  "homepage": "https://github.com/uphold/range-specifier-parser",
+  "repository": "git@github.com:uphold/range-specifier-parser",
   "license": "MIT",
-  "author": "Seegno",
-  "main": "dist/src",
-  "repository": "git@github.com:seegno/range-specifier-parser",
+  "author": "Uphold",
+  "main": "src/index.js",
   "scripts": {
-    "changelog": "github_changelog_generator --no-issues --header-label='# Changelog' --future-release=v$npm_config_future_release && sed -i '' -e :a -e '$d;N;2,4ba' -e 'P;D' CHANGELOG.md",
-    "lint": "eslint src test && jscs src test",
-    "test": "NODE_ENV=test mocha $npm_package_options_mocha",
-    "version": "npm run changelog --future-release=$npm_package_version && git add -A CHANGELOG.md"
+    "changelog": "uphold-scripts changelog $npm_package_version",
```

```suggestion
    "changelog": "uphold-scripts changelog v$npm_package_version",
```

Since this project seems to prefer to use the `v` prefix for the release names.